src/media/css/constants/colors.styl

```diff
@@ -1,20 +1,56 @@
+/* 
+ * colors.styl -- contains variables for each color used in CSS.
+ *
+ * Any time you would reference a color in CSS, you should instead be using a
+ * variable defined in either this file or a project-specific colors file. This
+ * allows for consistency and predictability in color definition and use.
+ * 
+ * Colors are grouped into palettes related by intention or purpose. It is
+ * encouraged for palettes to reference other palettes' colors. For example,
+ * if the active navigation item's underline uses the primary palette's orange,
+ * you might define it this way:
+ * 
+ * $color--navigation--active-underline = $color--primary--orange;
+ *
+ * This provides flexibiltiy in changes to our use of colors; if we want to
+ * globally redefine the orange color in the primary palette,
+ * $color--primary--orange can be redefined. If we only want to change the
+ * active navigation underline color, $color--navigation--active-underline can
+ * be changed.
+ *
+ * Variable names should follow the form $color--pallete-name--color-name
+ */
+
 // Primary palette
-$primary-blue = #3A5990;
-$primary-teal = #00C9A8;
-$primary-green = #8BC873;
-$primary-yellow = #FAD73B;
-$primary-orange = #ED8040;
-$primary-red = #D46952;
-$primary-pink = #E77286;
-$primary-purple = #8D4B87;
+$color--primary--blue = #3A5990;
```

Comment:
I'm going to go through this with pwalm to update current thinking once he's free of desktop work, don't worry about the accuracy of colors for now.